processor/groupbytraceprocessor/factory_test.go

```diff
@@ -39,9 +37,6 @@ func TestDefaultConfiguration(t *testing.T) {
 func TestCreateTestProcessor(t *testing.T) {
 	c := createDefaultConfig().(*Config)
 
-	logger, err := zap.NewDevelopment()
-	require.NoError(t, err)
-
 	params := component.ProcessorCreateParams{
 		Logger: logger,
```

Comment:
It's not apparent to me what this logger is now

Reply:
This is defined in the `processor_test.go` as:

```go
var (
	logger, _ = zap.NewDevelopment()
)
```